backend/ai_service.py

```
@@ -0,0 +1,364 @@
+            response = self._clean_response(response)
+            
+            return response if response else "I understand. How can I help you further?"
+            
+        except Exception as e:
+            logger.error(f"Chat generation failed: {e}")
+            return "I'm here to help! Could you tell me more about what you'd like to discuss?"
+    
+    def _create_system_prompt(self, emotion: Dict, sentiment: Dict) -> str:
+        """Create system prompt based on detected emotion and sentiment"""
+        emotion_label = emotion.get("label", "neutral").lower()
+        sentiment_label = sentiment.get("label", "NEUTRAL").lower()
```

Ruff (0.11.9)

Local variable `sentiment_label` is assigned to but never used

Remove assignment to unused variable `sentiment_label`

(F841)